lib/mat_SDCZ.mli

```diff
@@ -52,6 +52,15 @@ val to_array : mat -> num_type array array
 (** [to_array mat] @return an array of arrays initialized from matrix
     [mat]. *)
 
+val of_list : num_type list list -> mat
+(** [of_list ar] @return a matrix initialized from the list of lists
+    [ar]. It is assumed that the OCaml matrix is in row major order
+    (standard). *)
```

Comment:
Lacaml matrices use FORTRAN layout and so are column major.  What should be described is how the `num_type list list` is presented — that may be what you meant, but it is not clear.